src/MainWindow.java

```diff
@@ -183,43 +197,47 @@ public static void main(String[] args) throws InterruptedException {
 		}
 	}
 
-	public void saveGame() throws FileNotFoundException {
-		PrintWriter printWriter = new PrintWriter("savedGame.txt");
-
-		//saves the board state
-		for (int i = 0; i < gameWindow.board.H; i++) {
-			for (int j = 0; j < gameWindow.board.W; j++) {
-				printWriter.print(gameWindow.board.board[i][j] + " ");
-			}
+	public boolean saveGame() {
```

Comment:
Use a better order so that we can pass everything to `.load()` and avoid spamming syscalls, also try-catch instead of throwing exceptions because this isn't competitive

Reply:
We don't like exceptions